packages/components/src/particles/Banner/index.tsx

```diff
@@ -46,13 +53,13 @@ export default function Banner(props: BannerProps) {
       'text-red-600': props.status === 'error',
       'text-green-600': props.status === 'success',
       'text-yellow-600': props.status === 'warning',
-      'text-gray-600': props.status === 'suggestion'
+      'text-slate-600': props.status === 'suggestion'
```

Comment:
there's only one section using this setting and it's for the popup when acknowleding alerts. making it darker to help it pop a lil bit more.